core/src/com/unciv/logic/map/tile/TileInfoImprovementFunctions.kt

```diff
@@ -277,7 +277,9 @@ class TileInfoImprovementFunctions(val tile: Tile) {
 
         if (RoadStatus.values().any { improvementName == it.removeAction }) {
             tile.removeRoad()
-        } else {
+        }
```

Comment:
I think this looks fine, tbh 🙃

Reply:
I mean I see where Trog is coming from, technically the else should be on the same line of the trailing brace as per kotlin's coding conventions (I started more on C++, so I still am used to what's more common there and wasn't aware), Am imo it would look weird to move this else up to the prior line, but not use braces for line 281/282 here. Though, I kinda feel like this if/else chain maybe makes more sense as a `when` (both statements are 1 line, I don't really think it's fully sensible turn line 281 into a multiline thing when it doesn't even hit the 100 character threshold one may expect to split it into multiple lines, and 279 is only a multiline statement because the condition itself is long)